glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: b2ac93355c3f551a75216a800337cee9321f6c9a04a18ab1fa8d8152e89b7595
```

Comment:
In particular, go-marathon's version is specified as `^0.5.1` in glide.yaml, which is still valid given the latest version is 0.7.1.